packages/cli/src/fulltext/querySymbols.ts

```diff
@@ -25,6 +27,18 @@ const genericSymbolRegex =
   SymbolRegexes[ext] = SymbolRegexes.cpp;
 });
 
+function expandSymbol(symbol: string): string[] {
+  const results = sanitizeKeyword(symbol)
+    .flatMap((t) =>
+      splitCamelized(t)
+        .split(/[\s-_]/)
+        .map((s) => s.toLowerCase())
+    )
+    .filter((t) => t.trim().length > 1);
+  const unique = new Set([...results, symbol.toLowerCase()]);
+  return Array.from(unique).filter((t) => !STOP_WORDS.has(t));
+}
```

Comment:
I think this happens when the words are added to the index, so does it also need to happen here?